dask/dataframe/core.py

```diff
@@ -590,15 +590,13 @@ def random_split(self, frac, random_state=None):
         --------
         dask.DataFrame.sample
         """
-        seeds = different_seeds(self.npartitions, random_state)
-        dsk_full = dict(((self._name + '-split-full', i),
-                         (pd_split, (self._name, i), frac, seed))
-                        for i, seed in enumerate(seeds))
-
-        dsks = [dict(((self._name + '-split-%d' % i, j),
-                      (getitem, (self._name + '-split-full', j), i))
-                     for j in range(self.npartitions))
-                for i in range(len(frac))]
+        state_data = random_state_data(self.npartitions, random_state)
+        name = self._name + '-split-full'
```

Comment:
Is there a reason to continue the previous name here?  Perhaps just `name='split'`?